recipes/example/meta.yaml

```diff
@@ -1,83 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Can you restore the example

Reply:
Restored in 0529a715a86e178f8f9bdb94a3ff01ad0374474c